### PR TITLE
Fortran books #5492

### LINKED ISSUE #5492: Fortran books
I suggest adding a link to the tutorials section of the Fortran Wiki http://fortranwiki.org/fortran/show/Tutorials and to 

Modern Fortran in Science and Technology by Jonas Lindemann and Ola Dahlblom
Introduction to Advanced Fortran
Introduction to Fortran
Modern Fortran Tutorial
Fortran Lecture notes from a class by Paul Tackley

which are linked there. Arguably the Fortran 77 book currently listed should be removed, since it is about an antiquated standard.

### PR DESCRIPTION
**Pull request by davorpa**
## What does this PR do?
Add resource(s) | Add info

### Description
Add some Fortran books

### Why is this valuable (or not)?
Resolves #5492. All details there.

### How do we know it's really free?
Online resources freely distributable with acknowledgment

### For book lists, is it a book? For course lists, is it a course? etc.
In books section

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Followup

- Check the output of Travis-CI for linter errors!

### CODE REVIEW

File: books/free-programming-books.md [hunk 1]
```diff
@@ -1460,8 +1460,18 @@ Kerridge (PDF) (email address *requested*, not required)
 
 ### Fortran
 
-* [Professional Programmer’s Guide to Fortran77](https://www.star.le.ac.uk/~cgp/prof77.pdf) (PDF)
-* [Self Study Guide: Programming in Fortran 95](http://www.mrao.cam.ac.uk/~rachael/compphys/SelfStudyF95.pdf) (PDF)
+* [Fortran 90 Tutorial](http://www.cs.mtu.edu/~shene/COURSES/cs201/NOTES/fortran.html) - C.-K. Shene at Michigan Technological University (HTML)
+* [Fortran 90 Tutorial](https://web.stanford.edu/class/me200c/tutorial_90/) - Sarah T. Whitlock, Paul H. Hargrove at Stanford University (HTML)
+* [Fortran information & resources](https://www.fortranplus.co.uk/fortran-information/) - fortranplus.co.uk (HTML)
+* [FORTRAN Performance Tuning co-Guide (1998)](https://www.ibiblio.org/pub/languages/fortran/unct.html) - Timothy C. Prince (HTML)
+* [Fortran Wiki - Tutorials](http://fortranwiki.org/fortran/show/Tutorials) (HTML)
```
Comment (eshellman) [inline books/free-programming-books.md:1467]: not a book

File: books/free-programming-books.md [hunk 2]
```diff
@@ -1460,8 +1460,18 @@ Kerridge (PDF) (email address *requested*, not required)
...
+* [Fortran. Lecture notes from a class (2020)](http://jupiter.ethz.ch/~pjt/fortran/FortranClass.html) - Paul Tackley (HTML)
```
Comment (eshellman) [inline books/free-programming-books.md:1468]: not a book

**Comment by eshellman** [comment]
apart from the 2 I flagged, these are great additions

File: books/free-programming-books.md (hunk 1 above)
Comment (davorpa) [reply books/free-programming-books.md:1467]: What line exactly? 1467?

File: books/free-programming-books.md (hunk 2 above)
Comment (davorpa) [reply books/free-programming-books.md:1468]: Line 1468?

**Comment by eshellman** [comment]
weird. yes, 1467 and 1468

**Comment by davorpa** [comment]
> weird. yes, 1467 and 1468

Done

**Pull request merged**